inlong-sort/sort-connectors/base/src/main/java/org/apache/inlong/sort/base/format/DebeziumJsonDynamicSchemaFormat.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.inlong.sort.base.format;
+
+import org.apache.flink.shaded.jackson2.com.fasterxml.jackson.databind.JsonNode;
+
+/**
+ * Debezium json dynamic format
+ */
+public class DebeziumJsonDynamicSchemaFormat extends JsonDynamicSchemaFormat {
+
+    private static final String IDENTIFIER = "debezium-json";
+
+    private static final DebeziumJsonDynamicSchemaFormat FORMAT = new DebeziumJsonDynamicSchemaFormat();
+
+    private DebeziumJsonDynamicSchemaFormat() {
+
+    }
+
+    @SuppressWarnings("rawtypes")
+    public static AbstractDynamicSchemaFormat getInstance() {
+        return FORMAT;
+    }
+
+    @Override
+    protected JsonNode getPhysicalData(JsonNode root) {
+        return root.get("after");
```

Comment:
How to get topic name  when `op`  is delete data.

Reply:
done